eng/dockerfile-templates/sdk/Dockerfile.windows.install-components

```
@@ -71,7 +71,7 @@ RUN powershell -Command " `
                 "dotnet-is-internal": ARGS["dotnet-is-internal"]
             ],
             "        ")}}{{
-if !(dotnetVersion = "3.1" || isServerCore): `
+if !(dotnetVersion = "3.1" || isServerCore) || ARGS["dotnet-is-internal"]: `
```

Would it make sense to utilize isSingleStage here?  It limits the use of `dotnet-is-internal` and seems a little more readable as that is the reason shutdown has to be invoked.

```suggestion
if dotnetVersion != "3.1" || isSingleStage : `
```